grammar.js

```diff
@@ -819,7 +823,7 @@ module.exports = grammar({
 			choice($.simple_identifier, "class")
 		),
 
-		_assignment_and_operator: $ => choice("+=", "-=", "*=", "/=", "%="),
+		_assignment_and_operator: $ => choice("+=", "-=", "*=", "/=", "%=", "="),
```

Comment:
If you look at the official grammar here https://kotlinlang.org/docs/reference/grammar.html, assignment is written as
```
assignment (used by statement)
  : directlyAssignableExpression '=' expression
  | assignableExpression assignmentAndOperator expression
  ;
```
so maybe we should not touch _assignment_and_operator but add a case for just '=' below:
```
		assignment: $ => choice(
			prec.left(PREC.ASSIGNMENT, seq($.directly_assignable_expression, $._assignment_and_operator, $._expression)),
			// TODO
		),
```
Maybe just choice('=',$._assignment_operator) for now.